secure_message/common/alerts.py

```diff
@@ -27,18 +27,18 @@ def send(email, reference):
                                  json=notification)
 
         if response.status_code != 201:
-            raise RasNotifyException(code=500)
+            raise RasNotifyException(code=500, survey_id=survey_id, party_id=party_id)
         else:
             logger.info('Sent secure message email notification, via RM Notify-Gateway to GOV.UK Notify.',
-                        message_id=response.json()["id"])
+                        message_id=response.json()["id"], survey_id=survey_id, party_id=party_id)
 
 
 class AlertViaLogging:
     """Alert goes via gov notify (0) or via logs (1)"""
 
     @staticmethod
-    def send(email, reference):
-        logger.info(f'email details {email}, {reference}')
+    def send(email, msg_id, survey_id, party_id):
+        logger.info(f'email details email={email}, msg_id={msg_id}, survey={survey_id} and party_id={party_id}')
```

Comment:
Also this should use key/value pair logging
`logger.info("Email sent", msg_id={msg_id}, survey={survey_id}, party_id={party_id})`